conda/zipline/meta.yaml

```diff
@@ -1,28 +1,37 @@
+{% set data = load_setuptools() %}
+
 package:
   name: zipline
-  version: 0.8.3
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '')}}
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
 
 source:
-  fn: zipline-0.8.3.tar.gz
-  url: https://pypi.python.org/packages/source/z/zipline/zipline-0.8.3.tar.gz
-  md5: 042ffcee614d2279add9a1bfd27a33cf
+  git_url: ../../
 
 requirements:
   build:
     - python
     - setuptools
-    - cython
+    - cython ==0.22.1
+    - numpy ==1.9.2
```

Comment:
With numpy>=1.10 I'm getting `ValueError: numpy.dtype has the wrong size, try recompiling`.  I think @llllllllll  was getting this too.  

`numpy` also included in the `run` requirements, so I'll remove this pinned version, but add the cap at 1.9 for now.  I'll also give us back non-strict versions on the other run requirements - that was unintentional.